Microsoft.DotNet.Interactive.FSharp/FSharpPocketViewTags.fs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Interactive.FSharp
+
+open Microsoft.DotNet.Interactive.Rendering
+
+type FSharpPocketViewTags(p: PocketView) as this =
+    override __.ToString() = p.ToString()
+    abstract member HTML: obj -> FSharpPocketViewTags
+    abstract member Item: string * obj -> FSharpPocketViewTags with get
+    default __.HTML content =
+        p.SetContent([|content|])
```

Comment:
Ah, I think I see the confusion.

I'm simply trying to get around the fact that an F# object can't be directly invoked.  E.g., this doesn't work:

``` f#
div.["class", "c"]("body")
//                ^^^^^^^^ this fails because
//  ^^^^^^^^^^^^^^         this isn't a function
```

So we have to have an actual function that _can_ be invoked:

``` f#
div.["class", "c"].HTML("body")
//                 ^^^^ this IS a function and it CAN be invoked
```

Originally I called this `innerHTML` to mimic the HTML DOM, but then I found `HTML()` on `PocketViewTags` and thought I'd mimic that.  Ultimately I don't care what this is called; the only restriction is that it has to exist.

Is there a name you'd prefer, or is `HTML` (or `innerHTML`) fine?

Reply:
Spoke offline with @jonsequitur and @TIHan and long story short I mis-read the meaning of the `HTML()` method.  To try to make the meaning clear I'm copying the DOM method name `innerHTML()`.

There were other discussions about utilizing a constructor to set the inner content, but that opens a whole new can of worms around using the helper effectively.